kubernetes/windows/main.ps1

```diff
@@ -39,6 +39,50 @@ function Start-FileSystemWatcher {
     Start-Process powershell -NoNewWindow .\filesystemwatcher.ps1
 }
 
+function Set-AMAEnvironmentVariables {
+    
+    [System.Environment]::SetEnvironmentVariable("MONITORING_DATA_DIRECTORY", "C:\\opt\\windowsazuremonitoragent\\datadirectory", "Process")
```

Comment:
instead of having duplicate code for "Process" & "Machine". probably wrapping to smaller helper function which takes env & value would be cleaner.